test/CadlRanchProjects.Tests/type-union.cs

```diff
@@ -20,13 +20,13 @@ public class TypeUnionTests : CadlRanchTestBase
         {
             var response = await new UnionClient(host, null).GetStringsOnlyClient().GetStringsOnlyAsync();
             Assert.AreEqual(200, response.GetRawResponse().Status);
-            Assert.AreEqual(GetResponseProp.B, response.Value.Prop);
```

Comment:
This is a unknown issue. Anonymous model naming sequence is undefined and could be random.